kpatch-build/create-diff-object.c

```diff
@@ -1568,7 +1596,11 @@ static void kpatch_replace_sections_syms(struct kpatch_elf *kelf)
 				end = sym->sym.st_value + sym->sym.st_size;
 
 				if (!is_text_section(sym->sec) &&
+#ifdef __x86_64__
 				    rela->type == R_X86_64_32S &&
+#elif defined(__aarch64__)
+				    rela->type == R_AARCH64_ABS64 &&
+#endif
```

Comment:
No need to '#ifdef' this, the defines exist in /usr/include/elf.h regardless of arch.

Reply:
I guess what @jpoimboe is saying is that it can be written down as `(rela->type == R_X86_64_32S || rela->type == R_AARCH64_ABS64)` which looks nice. We also might want to get rid of all ifdefs at some point (see #1179) so unless it is necessary we are reluctant to add new ones.

Reply:
Ah yes, that makes sense